Exercise_A/index.html

```diff
@@ -5,32 +5,142 @@
     <!--Allows us to track the progress of the scroll -->
     <!-- Nav bar items -->
+    <script src="/Exercise_A/assets/js/wow.js"></script>
+    <script>
+    new WOW().init();
+    </script>
 
     <!-- Can use Jumbotron -->
     <!-- Second Section -->
+    <div class="second-section">
+      <img src="/Exercise_A/assets/images/header.jpg" alt="">
+      <div class="second-section-text">
+        <h1>
+          HEY LOOK, IT'S EVERY BOOTSTRAP WEBSITE EVER
+        </h1>
+        <hr>
+        <p>
+          Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all the same things as those pages!
+        </p>
+        <a data-scroll href="#four-icons">
+          <button id="scroll-to-icon">
+            THIS BUTTON MAKES THE PAGE SCROLL
+          </button>
+        </a>
+      </div>
+    </div>
 
     <!-- Four icons -->
+    <div class="four-icons-container" id="four-icons">
+      <h3>
+        So Fucking Creative
+      </h3>
+      <hr>
+      <div class="row four-icons">
+        <div class="col-12 col-md-6 col-lg-3 icon-container">
+          <i
+            class="fa-solid fa-earth-americas"
+            style="font-size: 50px; color: #ff5d39; margin: 0.5rem 0;"
+          ></i>
+          <h5>
+            This Fucking Template
+          </h5>
+          <p>
+            Guaranteed to use the same fucking template that every other bootstrap website uses, downloaded straight from The Web™
+          </p>
+        </div>
+        <div class="col-12 col-md-6 col-lg-3 icon-container">
+          <i
+            class="fa-solid fa-paper-plane"
+            style="font-size: 50px; color: #ff5d39; margin: 0.5rem 0;"
+          ></i>
+          <h5>
+            This Set of Four Icons
+          </h5>
+          <p>
+            Look at this cool set of four icons describing different things about us! We use four, because it's the default.
+          </p>
+        </div>
+        <div class="col-12 col-md-6 col-lg-3 icon-container">
+          <i
+            class="fa-solid fa-thumbs-up"
+            style="font-size: 50px; color: #ff5d39; margin: 0.5rem 0;"
+          ></i>
+          <h5>
+            Lots of effort
+          </h5>
+          <p>
+            We even changed some of the icons! We take the extra effort to make our designs truly original.
+          </p>
+        </div>
+        <div class="col-12 col-md-6 col-lg-3 icon-container">
+          <i
+            class="fa-solid fa-heart animate__animated animate__heartBeat animate__fast animate__infinite"
+            style="font-size: 50px; color: #ff5d39; margin: 0.5rem 0;"
+          ></i>
+          <h5>
+            Made with Love
+          </h5>
+          <p>
+            Because nothing says hard work and talent like editing a few lines of text.
+          </p>
+        </div>
+      </div>
+    </div>
     <!-- Four pictures with overlay etc -->
+    <div class="four-pics row">

```

Comment:
Great work here with the icons and adding in the animations using the Animate.CSS Library